src/lib/AbstractEvent.ts

```diff
@@ -11,36 +11,36 @@ let _callListenerResult = CallListenerResult.NONE;
  * should not be instantiated but extended by a specific event class. For an event class with basic
  * functionality that can be instantiated see [[BasicEvent]]
  */
-abstract class AbstractEvent implements IEvent
-{
+abstract class AbstractEvent implements IEvent {
 	/**
 	 * Will be updated by [[EventDispatcher]] during the dispatch of an event to the target that
 	 * listeners are currently being called on. After completion of an event dispatch this value
 	 * will be reset to _null_.
 	 */
-	public currentTarget:IEventDispatcher = null;
+	public currentTarget: IEventDispatcher = null;
 	/**
 	 * Will be updated by [[EventDispatcher]] when [[EventDispatcher.dispatchEvent|dispatchEvent]] is
 	 * called with this event. The value will be set to the EventDispatcher instance that dispatched
 	 * the event.
 	 */
-	public target:IEventDispatcher = null;
+	public target: IEventDispatcher = null;
 	/**
-	 * The current event phase of this event. During event dispatch, this value will be either [[EventPhase.CAPTURING_PHASE|CAPTURING_PHASE]],
-	 * [[EventPhase.AT_TARGET|AT_TARGET]] or [[EventPhase.BUBBLING_PHASE|BUBBLING_PHASE]]. If this event is not currently
-	 * being dispatched this will be set to [[EventPhase.NONE|NONE]].
+	 * The current event phase of this event. During event dispatch, this value will be either
+	 * [[EventPhase.CAPTURING_PHASE|CAPTURING_PHASE]], [[EventPhase.AT_TARGET|AT_TARGET]] or
+	 * [[EventPhase.BUBBLING_PHASE|BUBBLING_PHASE]]. If this event is not currently being dispatched this will be
+	 * set to [[EventPhase.NONE|NONE]].
 	 */
-	public eventPhase:EventPhase = EventPhase.NONE;
+	public eventPhase: EventPhase = EventPhase.NONE;
 	/**
 	 * Indicates the time this event is dispatched in the number of milliseconds elapsed since
 	 * _1 January 1970 00:00:00 UTC_. This value will only be set if the setTimestamp parameter in the constructor
 	 * is set to _true_. Otherwise, this value will be _0_.
 	 */
-	public timeStamp:number;
+	public timeStamp: number;
 	/**
 	 *  _true_ if [[cancelable]] is true and [[preventDefault]] has been called on this event.
 	 */
-	private _defaultPrevented:boolean = false;
+	private _defaultPrevented: boolean = false;
```

Comment:
Does the airbnb standard not force a private variable to be prefixed without a _

Reply:
@jesse-mm The MediaMonks coding standards suggest private variables to be prefixed with the "_", see the [coding standard document](https://docs.google.com/document/d/1D2EPaZqd6bGetsDjdx4EBSDMkNBuahp86UVG9xbFvAk/edit#heading=h.z337ya). I noticed more differences between the coding standards and our airbnb standards though, did we decide on dropping document entirely?

Reply:
That doc is a bit old-ish, for newer projects (react, vue) we switched to the air-bnb coding standard to keep everything consistent between our own (both js and ts) projects, and possibly others that will be using our code.

Reply:
So when did we decide what rules to overwrite in the `tslint.json` file? 

I already had some issues with node modules where the import name rule results in errors, see the seng-disposable example in the readme:
```ts
import Disposable from 'seng-disposable';
...
class AsyncThinger extends Disposable {
...
```
this fails with the current default tslint values. Also when doing for loops for example you cannot use the default `itar` live template anymore since it fails in our standards which is kind of annoying.